### PR TITLE
Remove legacy context

### LINKED ISSUE #596: Warning: Legacy context API has been detected within a strict-mode tree.
Warning: Legacy context API has been detected within a strict-mode tree.

The old API will be supported in all 16.x releases, but applications using it should migrate to the new version.

Please update the following components: ReactSwipableView

Learn more about this warning here: https://fb.me/react-legacy-context
    in ReactSwipableView

- [x] I have searched the [issues](https://github.com/oliviertassinari/react-swipeable-views/issues) of this repository and believe that this is not a duplicate.

| Tech                   | Version |
|------------------------|---------|
| react-swipeable-views  | 0.13.9 |
| React                  | 16.13.1  |
| platform               |     Win 10    |
| etc                    |         |

### PR DESCRIPTION
**Pull request by ykzts**
Follow-up #631 

Closes #596

### CODE REVIEW

File: docs/src/pages/demos/DemoAnimateHeight.js [hunk 1]
```diff
@@ -35,12 +34,15 @@ for (let i = 0; i < 30; i += 1) {
 }
 
 class Slide4 extends PureComponent {
+  static contentType = swipeableViewsContext;
```
Comment (korbav) [inline docs/src/pages/demos/DemoAnimateHeight.js:37]: Shouldn't it be `contextType` instead of `contentType` ?

Comment (ykzts) [reply docs/src/pages/demos/DemoAnimateHeight.js:37]: oh

Comment (ykzts) [reply docs/src/pages/demos/DemoAnimateHeight.js:37]:
```suggestion
  static contextType = swipeableViewsContext;
```

**Review by korbav — COMMENTED** [review]
Looks good to me!

**Comment by marklai1998** [comment]
any update on this?

**Comment by korbav** [comment]
> 
> 
> any update on this?

Maybe @oliviertassinari could take a look and merge it if it's suitable as it seems to be.

**Review by oliviertassinari — APPROVED** [review]

**Comment by marklai1998** [comment]
@ykzts merge it?

**Comment by ykzts** [comment]
@marklai1998  I don't have permission to merge into this repository.

**Comment by pgrodrigues** [comment]
@oliviertassinari can you merge this one and release a new version with the recent updates? Thanks in advance.

File: packages/react-swipeable-views/src/SwipeableViews.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
```
Comment (IegorShumov) [inline packages/react-swipeable-views/src/SwipeableViews.js:1]: I think it can be `import React, { createContext } from 'react';`

**Pull request merged**